Project-1/README.md

```diff
@@ -11,56 +11,276 @@ Tip #4: The work we do is challenging; that should be assumed. You are smart eno
+
   (d)  O that says that x is an only child  
+
+  O ≡ ¬ ∃p2¯.Per
+
   (e)  T that says that x has exactly two brothers 
 
+  This cannot be done without a role name for brother, we only have a concept name for it.
+
+  ```
 
 5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+	
+	TBox = {
+B ⊑ M
+B ⊑ Per
+M ⊑ Per
+A ⊑ ¬M
+A ⊑ Per
+}
+
+ABox = { John : M
+John : parentof. Chris
+John: B
+Chris: M
+Mary : ¬M
+Mary: A
+Alexander : parentof. Mary
+Alexander : parentof. John
+}
+
+ΔI = {John, Mary, Alexander, Chris}
+
+Named Individuals:
+JohnI = J,
+MaryI = M,
+AlexanderI = A,
+Chris = C
+
+Concept Assignments:
+PersonI = {J, M, A, C},
+MaleI = {J, A, C},
+FemaleI = {M},
+BrotherI = {J},
+AuntI = {M}
+
+Role Assignments:
+parentofI = {(J, C), (A, J), (A, M)}
+	
+	
 
 6. Explain the difference - using natural language - between the first-order prefixes:
-  ```(a) ∃x∀y and ∀x∃y
+  ```
+  (a) ∃x∀y and ∀x∃y
+  	** These read "There exists some X such that for all y..." and "For all x there exists some y such that..." There is no way to explain the difference beyond this because these are incomplete sentences. They are analogous to "sentence fragments" from natural language where the Subject was missing and the only verb is "is"; "Is large and blue."  **
+	
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  	** These read "There exists some X such that for all Y and some z..." and "For all X there exists some Y such that all Z. .. " There is no way to explain the difference beyond this because these are incomplete sentences **
+	
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  	** These read "For all X there exists some Y such that all Z and some W. . ." and "There exists some X for all Y such that some some Z and all W . . . " There is no way to explain the difference beyond this because these are incomplete sentences.  **
+  
 ```
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+```
+ {(a,b)(b,a)(a,a)} 
+ The second statement is true in this model. There is an x that relates every y (a, related to a and b) There is an x that relates to a z (where a is x and b is z), all ys (a and b) are related to some z (a is related to a and b, b is related to a)
+ The first statement is not true. It is not true that all xs (a and b) are related to all zs (b is not related to a)
+
 ```
 	
 8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
-  ```(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  ```
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
```

Comment:
What explains the move in line 1 from a negated universal to line 2 with a negated universal?